org.eclipse.jdt.core.manipulation/common/org/eclipse/jdt/internal/ui/fix/MultiFixMessages.properties

```diff
@@ -193,3 +193,5 @@ StringBufferToStringBuilderCleanUp_description=Convert StringBuffer to StringBui
 StringConcatToTextBlockCleanUp_description=Convert String concatenation to Text Block
 StringConcatToTextBlockStringBuffer_description=Convert String/StringBuffer/StringBuilder concatenation to Text Block
 StringBuilderForLocalVarsOnlyCleanUp_description=Convert StringBuffer to StringBuilder for local variables
+ExplicitEncodingCleanUp_description=Set explicit encoding or default encoding where applicable on methods ''{0}'' using {1}
```

Comment:
This message is too long in the configure cleanups view.  Drop the list of methods.

Reply:
Done